<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Subway Guide</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/resources/icon/subway.png</normaloff>:/resources/icon/subway.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>275</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Regular Mode</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QWidget" name="l2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="SPButton">
             <property name="text">
              <string>最短路径</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="LTButton">
             <property name="text">
              <string>最少换乘</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QWidget" name="l3" native="true">
          <layout class="QFormLayout" name="formLayout_4">
           <item row="1" column="0">
            <widget class="QLabel" name="DistanceLable">
             <property name="text">
              <string>路程(km):</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="ShowDistance">
             <property name="text">
              <string>0.00</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="ShowFare">
             <property name="text">
              <string>0.00</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="FareLabel">
             <property name="text">
              <string>票价(¥):</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="TimeCostLable">
             <property name="text">
              <string>预计用时(min):</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="ShowTimeCost">
             <property name="text">
              <string>0.00</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="l1" native="true">
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="StartLabel">
             <property name="text">
              <string>起点：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="StartBox">
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="currentText">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="EndLabel">
             <property name="text">
              <string>终点：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="EndBox">
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="currentText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QTextBrowser" name="ShowRoute"/>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="ConfirmButton">
          <property name="text">
           <string>查询路线</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="PayButton">
          <property name="text">
           <string>支付</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="RouteLabel">
          <property name="text">
           <string>路线：</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Travel Mode</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QPushButton" name="ConfirmButton_T">
          <property name="text">
           <string>查询路线</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="RouteLabel_T">
          <property name="text">
           <string>路线：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="PayButton_T">
          <property name="text">
           <string>支付</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QTextBrowser" name="ShowRoute_T"/>
        </item>
        <item row="1" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>249</width>
             <height>69</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="rowWrapPolicy">
             <enum>QFormLayout::DontWrapRows</enum>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="l4" native="true">
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="StartLabel_T">
             <property name="text">
              <string>起点：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="StartBox_T">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QWidget" name="l5" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QRadioButton" name="SPButton_T">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>最短路径</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="LTButton_T">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>最少换乘</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QWidget" name="l6" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="FareLabel_T">
             <property name="text">
              <string>票价(¥):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ShowFare_T">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="myGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionStation_Manage"/>
    <addaction name="actionEdge_Manage"/>
    <addaction name="actionTourist_Attraction_Manage"/>
    <addaction name="separator"/>
    <addaction name="actionLog_in"/>
    <addaction name="actionLog_out"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionIntroduction"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>关于(&amp;A)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLog_in"/>
   <addaction name="actionLog_out"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStation_Manage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>站点配置文件管理</string>
   </property>
  </action>
  <action name="actionEdge_Manage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>边配置文件管理</string>
   </property>
  </action>
  <action name="actionLog_in">
   <property name="text">
    <string>管理员登录</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionShow_Tool_Bar">
   <property name="text">
    <string>Show Tool Bar</string>
   </property>
  </action>
  <action name="actionShow_Panel">
   <property name="text">
    <string>Show Panel</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于此项目</string>
   </property>
  </action>
  <action name="actionIntroduction">
   <property name="text">
    <string>程序介绍</string>
   </property>
  </action>
  <action name="actionLog_out">
   <property name="text">
    <string>管理员登出</string>
   </property>
  </action>
  <action name="actionTourist_Attraction_Manage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>景点配置文件管理</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">mygraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
